query/sources/common/src/main/java/org/codice/ddf/admin/sources/utils/RequestUtils.java

```diff
@@ -162,7 +160,7 @@ public ReportImpl endpointIsReachable(UrlField urlField) {
             LOGGER.debug("Successfully reached {}.", urlField);
         } catch (IOException e) {
             LOGGER.debug("Failed to reach {}, returning an error.", urlField, e);
-            report.addArgumentMessage(cannotConnectError(urlField.path()));
+            report = Reports.from(cannotConnectError(urlField.path()));
```

Comment:
This could be `return Reports.from(cannotConnectError(urlField.path()))`. And then move line 154 to 174 as a return.